docs/learn/expert-guides/universal-profile/create-profile.md

```diff
@@ -20,12 +20,6 @@ To easiest way to create a Universal Profile is through the Universal Profile Br
 - install the 🖥️ [Universal Profile Browser Extension](/install-up-browser-extension)
 - and ✨ [create a Universal Profile](https://my.universalprofile.cloud)
 
-However, if you want to create a profile programmatically you should use the ⚒️[lsp-factory.js](../../../tools/lsp-factoryjs/deployment/universal-profile.md). You can install it in your project using:
```

Comment:
Suggest moving this into a callout and linking to the LSP23 guide. (Need to change the link in my code example.

```
:::success Alternative guide

For developers looking for a guide to create a profile programmatically, see our guide [**Deploy a UP + its Key Manager using the LSP23 Factory**](./link/to/learn/expert-guides/deploy-UP-with-LSP23).

:::
```